### ISSUE TITLE
Accessibility issue

### ISSUE THREAD

**Issue by MariaAug** [microsoft-search/pnp-modern-search#1651]
**Version used**
4.5.0

**Describe the bug**
This issue is related to #1298 - Focus should remain in the search box after the text is cleared using the 'clear' button. Thanks for fixing this and the focus stays on the search box when the clear button is clicked. However, it is not calling out to the user in NVDA, that the text is cleared. Also, while using NVDA, it clears the text but user is not able to type any other search keyword until they press Enter. It is working as expected when we normally use the tab key without NVDA tool, search for a text, use arrow key, press clear button and I am able to enter the text in search box without having to press Enter key additionally.

**To Reproduce**

1. Add PnP search box web part and PnP search results web part on SharePoint page configured to show results from Local SharePoint Results.
2. Save the page
3. Open NVDA screen reader tool
4. Enter a keyword in the search box and start pressing tab key using NVDA screen reader tool.
5. Use arrow keys to access the clear button if not accessible via tab key.
6. On the clear text icon, press Enter. After pressing ''clear text'' button, the text is cleared from the search box and the focus remains on the search box.
7. However, it does not call out to the user that the text is cleared and the user is unable to enter another search term unless user presses Enter key. 

**Expected behavior**
The control remains in NVDA tool after user clears the text and hence it is not allowing the user to type anything in the search box again. If it is complex to fix this issue, is it possible to call out as 'Search news. Search text empty. Press Enter key and type the keyword that you would like to search' if the clear button is clicked?

**Screenshots**

**Desktop (please complete the following information):**
 - Chrome browser or any browser

**Additional context**

**Comment by wobba** [comment]
I might have done a fix for this one - you can either build the dev branch or wait for a 4.5.3 release which we are planning to verify.
